packages/generator/tests/acceptance/test_non_streaming_measurement_client.py

```diff
@@ -33,19 +36,45 @@ def test___measurement_plugin_client___measure___returns_output(
     assert measure_output == expected_measure_output
 
 
+def test___measurement_plugin_client___stream_measure___returns_output(
+    measurement_plugin_client_module: ModuleType,
+) -> None:
+    test_measurement_client_type = getattr(measurement_plugin_client_module, "TestMeasurement")
+    output_type = getattr(measurement_plugin_client_module, "Output")
+    expected_measure_output = output_type(
```

Comment:
Rename to `expected_output`. You originally used `measure_output` vs `output` to disambiguate between the output type and output instance.

Reply:
Sure, renamed to `expected_output`.